src/elements/pageFiltersDrawer.scss

```diff
@@ -0,0 +1,33 @@
+@import '../shared/styles/colors';
+
+.ui.page--filters_drawer {
+    &.ui.drawer .drawer-container-inner {
+        .drawer-close-button-container {
+            top: 16px;
+        }
+        .drawer-header {
+            background-color: $bkgd-light;
+            padding-top: 49px;
+            .title {
+                top: 16px;
+            }
+        }
+    }
+}
+
+@media only screen and (max-width: 767px) {
+    .ui.page--filters_drawer {
+        .drawer-container-inner {
+            padding-left: 11px !important; // sass-lint:disable-line no-important
+            padding-right: 11px !important; // sass-lint:disable-line no-important
+        }
+        .drawer-header .title {
+            padding-left: 11px !important; // sass-lint:disable-line no-important
+            padding-right: 11px !important; // sass-lint:disable-line no-important
+        }
+        .drawer-close-button-container {
+            right: 11px !important; // sass-lint:disable-line no-important
+            width: 16px !important; // sass-lint:disable-line no-important
```

Comment:
Lots of `!important`s here.  As I understand it, that's a good thing to flag as a potential code smell, and the mechanism of last resort.  So, I'm dutifully calling it out.  If there is a good case that `!important` really is required here, so be it.  Just want to ensure that everyone (mainly @morethanfire ) is aware of the reason why and is on board with the usage.

Reply:
@groberts314 I haven't worked on this file. Might be automatic es-lint fixing while saving it. @morethanfire might have idea about it.

Reply:
Interesting.  It does appear this file is already like this in `dev` branch.
I think this PR would be introducing this file to `release/5.1.1` branch where it doesn't exist yet.  That's why it shows up as diffed.  My bad.  Then more than likely Señor Brewer is already well aware of the `!important`s. 😁

Reply:
These `!important`s are good.

Reply:
I guess they're, err, _important_ eh? 🤣